3529/2/Lib/Utiles/path_loader.rb

```diff
@@ -0,0 +1,15 @@
+require 'fileutils'
+
+class PathLoader
+  def initialize(user_id, chat_id)
+    @session_id = "#{user_id}:#{chat_id}"
+    @date = Time.now.strftime('%m:%d:%Y:%H')
+  end
+
+  def create_directory(in_or_out, file_name)
+    path = "public/#{@session_id}/check#{in_or_out}s/#{@date}/"
```

Comment:
https://ruby-doc.org/stdlib-2.4.1/libdoc/fileutils/rdoc/FileUtils.html#method-c-mkdir_p
Read please documentation or look into source code. You don't need this